conduct/HANDLING_CODE_OF_CONDUCT_REPORTS.md

```diff
@@ -15,57 +15,38 @@ Generally, reports to coc-escalation should refer to a report that was already s
-     1. Resolution shared: The resolution is shared with the reporter or target by the contact person. If the reporter or target expresses concerns then the contact person will bring these back to the CPC members for further discussion. This cycle can continue until the CPC members reach consensus that the reporter or target's concerns have been adquately addressed. If no feedback is received within 7 days the resolution is considered as accepted.
-     1. Final resolution: the resolution agreed by the CoCP members is implemented and the outcome reported back to the reporter or target bythe main contact.
+When a report is received to `report@openjsf.org` the following actions will be taken:
+   * An OpenJS Foundation staff representative will respond within 24 hours to the reporter or target confirming that the report has been received. 
+   * The staff representative will seek to determine whether the report is emergent and requires immediate attention (e.g. for an ongoing issue) or the report is a reactive response to an incident with no immediate danger. 
+
+   * The Staff Representative will identify two contact persons from its CoC Panel to examine and act on the report. The contact persons will be chosen on the basis of schedule and availability.
+   * The contact persons will then execute the following procedures:
+     1. Acknowledging the report: The contact persons will respond to let the reporter or target know that the report is being discussed.
+     1. Information gathering: The contact persons will collect information in the private moderation repository..
+     1. Discussing the report: The contact persons will discuss the facts of the report in context of opinions via meeting, email, or discussion via private issue.
+     1. Acting on the report: The contact persons will attempt to reach consensus on the action to be taken.
+     1. Sharing the resolution: If the contact persons are in agreement, the resolution is shared with the reporter or target by the contact persons and recorded in the moderation repository. If the contact persons are not in agreement or if the reporter or target expresses concern or dissatisfaction with the resolution,  then the contact persons will bring the report to the CoC Panel  for further discussion. This cycle can continue until the panel reaches consensus that the reporter or target's concerns have been adequately addressed. If the panel cannot reach consensus within 7 days, the matter will be taken to a vote of the panelists. If no feedback is received within 7 days the resolution is considered as accepted. 
+     1. Final resolution: the resolution is implemented and the outcome reported to the reporter or target, and/or other involved parties.
```

Comment:
Do I understand right that this is the first point at which a person accused of a CoC violation might hear about it? Or is communication with such a person implicit in the "Information gathering" phase?

I find it a little concerning that this process does not appear to allow for anyone to defend their actions before final resolution.